### PR TITLE
Fixed Message tests for Jetpack

### PR DESCRIPTION
**Pull request by hitesh96db**

### CODE REVIEW

**Comment by hitesh96db** [comment]
The changes on the extension side can be seen here - https://github.com/hitesh96db/privly-jetpack/commit/8483169c55fd12ebfe08d7bd3b930380a0c82ff3, which includes
* A content script to receive messages - `data/content_scripts/test/context_messenger.js`
* Injecting the content script when running integration tests - `lib/test_runner.js`

File: test/selenium/configurations.rb [hunk 1]
```diff
@@ -42,6 +42,7 @@ def common_configuration_for_firefox_extension
   xpi_filename = "@" + json['name'] + "-" + json['version'] + ".xpi"
   # Load the Firefox driver with the extension installed
   @profile = Selenium::WebDriver::Firefox::Profile.new
+  @profile["extensions.privly.integration_test"] = "true"
```
Comment (smcgregor) [inline test/selenium/configurations.rb]: What is this used for?

**Comment by smcgregor** [comment]
I think this looks good, with the one question I added.

@SummerWish Any comments?

**Comment by SummerWish** [comment]
Haven't tested it on Chrome but I guess it is good.

File: test/selenium/configurations.rb (hunk 1 above)
Comment (hitesh96db) [inline test/selenium/configurations.rb]: [Extension background script](https://github.com/privly/privly-jetpack/pull/11/files#diff-0d32a96b97e2c5e320d8df9cd42188e1R26) and [Test runner](https://github.com/privly/privly-jetpack/pull/11/files#diff-eb74954aff54d42f3dfa90cb1410ce87R54).
Injects the test content script(used for testing context messenger) only when the extension integration tests are run.

Comment (hitesh96db) [inline test/selenium/configurations.rb]: So that's used as a flag to signal that integration tests are being run.

**Comment by hitesh96db** [comment]
Resolved merge conflicts.

**Pull request merged**